src/App.tsx

```diff
@@ -17,40 +66,18 @@ export const App: React.FC = () => {
             type="text"
             placeholder="Enter a part of the name"
             className="input"
+            value={user}
+            onChange={event => handleQueryChange(event)}
+            onFocus={() => setIsFocused(true)}
+            onBlur={() => setTimeout(() => setIsFocused(false), 500)}
```

Comment:
do you really need a delay here? I'd remove it. Otherwise move the function to a separate variable